commitizen/providers/__init__.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import cast
+
+import importlib_metadata as metadata
+
+from commitizen.config.base_config import BaseConfig
+from commitizen.exceptions import VersionProviderUnknown
+
+from commitizen.providers.base_provider import VersionProvider
+from commitizen.providers.cargo_provider import CargoProvider  # noqa: F401
+from commitizen.providers.commitizen_provider import CommitizenProvider  # noqa: F401
+from commitizen.providers.composer_provider import ComposerProvider  # noqa: F401
+from commitizen.providers.npm_provider import NpmProvider  # noqa: F401
+from commitizen.providers.pep621_provider import Pep621Provider  # noqa: F401
+from commitizen.providers.poetry_provider import PoetryProvider  # noqa: F401
+from commitizen.providers.scm_provider import ScmProvider  # noqa: F401
```

Comment:
to prevent the noqa you could add the `__all__`:

```
__all__ = [
  "get_provider",
"VersionProvider",
"CargoProvider",
etc
]
```

Reply:
I used this approach instead. Hadn't thought of it, thank you!